flutter/lib/src/renderer/renderer.dart

```diff
@@ -0,0 +1,25 @@
+import 'unknown_renderer.dart'
+    if (dart.library.html) 'html_renderer.dart'
+    if (dart.library.io) 'io_renderer.dart' as implementation;
```

Comment:
Should we add a `noop` in case there's no `html` nor `io`, is that even possible?

Reply:
I don't think so, I just thought it would be better to be safe than sorry